app/views/sections/edit.html.erb

```diff
@@ -2,4 +2,6 @@
 
 <h1><%= yield :title %></h1>
 
+<h3>Course: <%= @course.name %></h3>
```

Comment:
Also, probably move it into the form partial, since its duplicated in the `new` template.